cfme/storage/volume.py

```diff
@@ -204,19 +222,44 @@ def delete(self, wait=True):
 
     @property
     def exists(self):
-        view = navigate_to(self.collection, 'All')
         try:
-            view.entities.get_entity(by_name=self.name, surf_pages=True)
+            navigate_to(self, 'Details')
             return True
-        except ItemNotFound:
+        except VolumeNotFound:
             return False
 
+    @property
+    def size(self):
+        view = navigate_to(self, 'Details')
+        return view.entities.properties.get_text_of('Size')
+
+    @property
+    def tenant(self):
+        view = navigate_to(self, 'Details')
+        return view.entities.relationships.get_text_of('Cloud Tenants')
+
 
 @attr.s
 class VolumeCollection(BaseCollection):
     """Collection object for the :py:class:'cfme.storage.volume.Volume'. """
     ENTITY = Volume
 
+    def create(self, name, storage_manager, tenant, size, provider):
+        """Create new storage volume"""
+        view = navigate_to(self, 'Add')
+        view.form.fill({'storage_manager': storage_manager,
+                        'tenant': tenant,
+                        'volume_name': name,
+                        'size': size})
+        view.form.add.click()
+        view.flash.assert_success_message('Cloud Volume "{}" created'.format(name))
```

Comment:
This flash message is for CFME 58z+, and needs to be version picked for 57z:
' Creating Cloud Volume "<name>"'